src/aggregator/aggregator/flush_mgr.go

```diff
@@ -404,7 +401,21 @@ func newBucket(
 			60 * time.Second,
 			90 * time.Second,
 			120 * time.Second,
-		}),
+			150 * time.Second,
```

Comment:
honestly i don't think these are necessary. 2m is terrible

Reply:
Yeah it's not great, but unfortunately it's not too uncommon :| 
Once we get some improvements in around this lag propagation it should be possible to ratchet this down